src/observePlugin.js

```diff
@@ -0,0 +1,15 @@
+import eventsPlugin from 'store/plugins/events';
```

Comment:
a link to the original file would be a great comment so people can see the source